src/index.html

```diff
@@ -11,7 +11,131 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="#" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">
+            Surname:
+            <input type="text" name="surname" id="surname"
+            required autocomplete="off">
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="name">
+            Name:
+            <input type="text" name="name" id="name"
+            autocomplete="off">
+          </label>
+        </div>
+        <div class="form-field">
+          How old are You?
+          <label for="age">
+            <input type="number" name="age" id="age"
+            value="12" min="1" max="100">
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="birth-date">
+            Full date of birth:
+            <input type="date" name="birth" id="birth-date">
```

Comment:
Check point 4 from checklist
```suggestion
            <input type="date" name="birth" id="birthDate">
```